### ISSUE TITLE
MDLabel font_name hitch

### ISSUE THREAD

**Issue by ClaesFredo** [kivymd/KivyMD#1576]
This is my first bug report - let me know where I go wrong. 

It is a minor thing and it has a workaround. Thought I should report if not for someone else to find the workaround. 

Problem description
The MDLabel test does not always correctly when Chinese, (Arabic, etc) fonts are used. 

lbl_1 (top) does not render as the font_name is not updated. (Kivy Label works as intended when defined as lbl_1 )

lbl_2 renders correctly when the font_name is updated after the MDLabel was defined. 

For what it is worth - it seems to be an updating problem. It is most certainly not a font problem as the problem appears for multiple fonts and languages. 

A font can be downloaded here: https://fonts.google.com/noto/specimen/Noto+Sans+SC 

![MDLabel_with_Chinese_font](https://github.com/kivymd/KivyMD/assets/12477840/1ef05d8f-4089-44a3-b66b-4afbfa4b87b8)

Code Example

```python
from kivymd.app import MDApp
from kivymd.uix.label import MDLabel
from kivy.uix.gridlayout import GridLayout

class DebugThis(MDApp):
    def build(self):        
        ch_fnt = './fonts/NotoSansSC-Regular.ttf'

        gl = GridLayout(cols=1)

        #Does not render as it should
        lbl_1 = MDLabel(
            text='中国人',
            font_context=None,
            font_name = ch_fnt
            )

        #OK
        lbl_2 = MDLabel(
            text='中国人',
            font_context=None,
            )
        lbl_2.font_name = ch_fnt

        gl.add_widget(lbl_1)
        gl.add_widget(lbl_2)

        return gl
    
DebugThis().run()
```

**Comment by HeaTTheatR** [comment]
Use fonts with Arabic/Chinese language support

**Comment by ClaesFredo** [comment]
As the example demonstrates - I do use a font with Chinese language support

The lbl_2 would not render correctly otherwise.

**Comment by HeaTTheatR** [comment]
This will not be fixed in version 1.2.0
Only in 2.0.0

**Comment by ClaesFredo** [comment]
That is fine for me. I am just trying to do what little I can to help by reporting that which I find.

**Comment by HeaTTheatR** [comment]
Most of the issue were deprecated along with previous versions of the KivyMD library. All these issue will be closed as soon as version 2.0.0 gets into the master branch.
